plugin-build/plugin/src/main/java/io/sentry/kotlin/multiplatform/gradle/plugin/CocoapodsAutoInstallExtension.kt

```diff
@@ -0,0 +1,32 @@
+package io.sentry.kotlin.multiplatform.gradle.plugin
+
+import org.gradle.api.Project
+import org.gradle.api.provider.Property
+import javax.inject.Inject
+
+@Suppress("UnnecessaryAbstractClass")
+abstract class CocoapodsAutoInstallExtension
+    @Inject
+    constructor(project: Project) {
+        private val objects = project.objects
+
+        /**
+         * Enable auto-installation of the Sentry Cocoa SDK pod.
+         *
+         * If the cocoapods plugin is applied and no existing Sentry pod configuration exists, the Sentry-Cocoa SDK pod will be installed
+         *
+         * Defaults to true.
+         */
+        val enabled: Property<Boolean> =
+            objects.property(Boolean::class.java)
+                .convention(true)
+
+        /**
+         * Overrides default Sentry Cocoa version.
+         *
+         * Defaults to the `~> 8.25`.
+         */
+        val sentryCocoaVersion: Property<String> =
+            objects.property(String::class.java)
+                .convention("~> 8.25") // todo: grab the latest version properly
```

Comment:
Could do it e.g. via `buildConfig` fields, similar to what we do for the sentry SDK in SAGP -  store it in `gradle.properties` and then retrieve like this https://github.com/getsentry/sentry-android-gradle-plugin/blob/f9b82ddce3af1a21eec54da8b1c02e36c063939a/plugin-build/build.gradle.kts#L348